torchrl/objectives/sac.py

```diff
@@ -450,9 +463,7 @@ def target_entropy(self):
             else:
                 action_container_shape = action_spec.shape
             target_entropy = -float(
-                action_spec[self.tensor_keys.action]
```

Comment:
This too I had to change. Shouldn't  action spec already have the correct shape?